src/main/java/org/folio/service/manager/inputdatamanager/reader/LocalStorageCsvSourceReader.java

```diff
@@ -22,8 +22,7 @@
 
   @Override
   public Iterator<List<String>> getFileContentIterator(FileDefinition fileDefinition, int batchSize) {
-    try {
-      Stream<String> lines = Files.lines(Paths.get(fileDefinition.getSourcePath()));
+    try(Stream<String> lines = Files.lines(Paths.get(fileDefinition.getSourcePath()))) {
```

Comment:
If to put Stream in _try_ block this will automatically close the Stream after reading the first partition/chunk. The previous implementation lefts the Stream unclosed, @illia-borysenko going to rework this part in his current task to close Stream in InputDataManager.finilzeExport method. 

@illia-borysenko  can you do a quick rework of on this branch, we should close the Stream anyway

Reply:
oh yea that's right, we should work on closing ht e stream, it'll be a memory leak otherwise

Reply:
Yes, it's good point, I can make and push a commit with rework to start closing stream to this branch